kiva/svg.py

```diff
@@ -413,7 +413,7 @@ def _emit(self, name, contents=None, kw={}, **otherkw):
 
     def _color(self, color):
         r,g,b,a = color
-        return '#%02x%02x%02x' % (r*255,g*255,b*255)
+        return '#%02x%02x%02x' % (int(r*255),int(g*255),int(b*255))
```

Comment:
This line looks odd to me, but that's orthogonal to this PR. If you want to scale the floating-point range `[0.0, 1.0]` to the integer range `[0, 256)`, doing it in such a way that only the exact value `1.0` maps to `255` and all other values map to something strictly smaller seems wrong.